### PR TITLE
DOC Add section on downloading CI built docs

### PR DESCRIPTION
**Pull request by lucyleeow**
# Description
Add section on downloading CI built docs
From https://github.com/napari/napari/pull/4944#issuecomment-1217226287  I thought some people may not know about this.
I understand that there has been talk/intention to improve how CI built doc artifacts are accessed (e.g., automatic preview) but before that happens I thought it would be worthwhile to add this to napari docs.

Trailing white space has also been removed from the `index.md` page, can add them back in if desired.

## Type of change

- [ ] Bug-fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# References

# How has this been tested?

- [ ] example: the test suite for my feature covers cases x, y, and z
- [ ] example: all tests pass with my change
- [ ] example: I check if my changes works with both PySide and PyQt backends
      as there are small differences between the two Qt bindings.  

## Final checklist:
- [ ] My PR is the minimum possible work for the desired functionality
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I included new strings, I have used `trans.` to make them localizable.
      For more information see our [translations guide](https://napari.org/developers/translations.html).

### CODE REVIEW

File: docs/developers/documentation/index.md [hunk 1]
```diff
@@ -136,7 +138,7 @@ subtrees:
 
 Once you've added your document to the `docs` folder and updated the `_toc.yml`, you can preview the website
  locally by running `make docs` from the root of
-the `napari` repository (assuming you've installed the [docs prerequisites](#prerequisites)).
+the `napari` repository (assuming you've installed the [docs prerequisites](prerequisites)).
```
Comment (Carreau) [inline docs/developers/documentation/index.md]:
```suggestion
the `napari` repository (assuming you've installed the [docs prerequisites](#prerequisites)).
```

I think the `#` is correct, it links to the eponymous section on line 6.

At least the links currently works when visiting https://napari.org/stable/developers/documentation/index.html#prerequisites

**Review by Carreau — DISMISSED** [review]
LGTM, just the `#` which I believe should be added back in.

File: docs/developers/documentation/index.md (hunk 1 above)
Comment (lucyleeow) [reply docs/developers/documentation/index.md]: Yes you are right, I missed it

**Comment by Czaki** [comment]
I think that going though checks tab is better as you could reach it from the top of PR page 

Select the check tab:

obraz

Click build Docs 

obraz

and download:

obraz


But this is my personal preference (work better on PR with long discussions) and requires less scrolling searching for a docs entry (normal PR has 30 checks) and is accessible without logging to GitHub.

**Comment by Carreau** [comment]
> I think that going though checks tab is better as you could reach it from the top of PR page

TIL !

**Review by melissawm — APPROVED** [review]
Looks great to me - I wonder if at some point we want to use CircleCI to actually expose the built docs. But this is already pretty good 👍🏻

**Comment by Carreau** [comment]
> I think that going though checks tab is better as you could reach it from the top of PR page

I'm going to merge this, and open an issue to decide whether to update to use this workflow.

**Comment by jni** [comment]
Hi folks! This is a periodic reminder that we're not supposed to merge pngs that are not managed by lfs. 🤦‍♂️ No worries, I think every core dev has made the mistake at least once. 😂 Hopefully we can put this all behind us once @melissawm finishes the migration to a separate docs repo. If review time is blocking here (as I suspect it has been), fyi I'll be back from holidays in ~2 days and eager to help push this through! (I'm currently in transit Europe->Aus.)

**Comment by Carreau** [comment]
Sorry, my bad, do you want me to revert/force-push ?

**Comment by jni** [comment]
No, history rewrite is pretty stressful. I suggest we let this one slide and once the docs migration is done, we can figure out whether a one-time cleanup is worth it.

**Pull request merged**